create_jenkins_job.py

```diff
@@ -77,7 +77,7 @@ def main(argv=None):
         'use_osrf_connext_debs_default': 'false',
         'use_fastrtps_default': 'true',
         'use_opensplice_default': 'false',
-        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON --',
+        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON',
```

Comment:
> Which causes the run_ros2_batch.py command-line parser to explode.

The `-- ---` doesn't make sense, but `--- --` would. So I'd say we need to fix the input, not the parsing in `run_ros2_batch.py`. But that's just my opinion without looking into the details very much.

In case you're wondering how this is handled in the build tool, see:

- https://github.com/ament/ament_tools/blob/master/ament_tools/verbs/build_pkg/cli.py#L67
- https://github.com/ament/ament_tools/blob/master/ament_tools/build_types/cmake.py#L93
- https://github.com/osrf/osrf_pycommon/blob/master/osrf_pycommon/cli_utils/common.py#L65-L163
- http://osrf-pycommon.readthedocs.io/en/latest/cli_utils.html#osrf_pycommon.cli_utils.common.extract_argument_group